system/install.sh

```diff
@@ -6,46 +6,34 @@ fi
 
 cd /
 echo Updating Debian
-apt-get update &&
+apt-get update
 apt-get upgrade -y
-apt-get install -y curl nano sudo git build-essential
+
+echo Installing Extras
+apt-get install -y curl nano git build-essential
```

Comment:
This should probably be in the install deps script. Or at least curl should be - it's not used in this script